#name : try { ... } catch { ... }
# --
try
{
  $0
}
catch(${1:type}Exception ${2:variable_name})
{
  $2.printStackTrace();
}